test/srv6-sighup-flow/check.sh

```diff
@@ -8,17 +8,21 @@ NC='\033[0m'
 echo -e "${C}RUNNING: $TEST_NAME${NC}"
 
 nohup ../../build.sh run $TEST_NAME -p $PORT_OPTIONS1 -c 1 --dur 5 &
-sleep 1
+# Extra time to load the signaler
+sleep 3
```

Comment:
This is also in https://github.com/williamofockham/NetBricks/pull/50 to fix the flapping. Whichever is reviewed first, I'll fix the conflict.